cockatrice/src/game/player/player.cpp

```diff
@@ -836,6 +839,8 @@ void Player::retranslateUi()
         sayMenu->setTitle(tr("S&ay"));
     }
 
+    aSelectAll->setText(tr("&Select All in Zone"));
```

Comment:
I don't think you intended to put the &S here, as that's an auto Windows hot-key.